collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -443,7 +432,7 @@ protected void updateAdapter() {
             filteredFormList.addAll(formList);
         }
         sortList();
-        formListAdapter.notifyDataSetChanged();
+        listView.setAdapter(new FormDownloadListAdapter(this, filteredFormList, formNamesAndURLs));
```

Comment:
@grzesiek2010 This seems odd. Could you notify an existing adapter of changes rather than building and then orphaning a new object on each character typed?